### PR TITLE
[hlm-707]: Created models for Delivery and Registration

### PR DESCRIPTION
**Pull request by jayantp-egov**
- Enhanced sample endpoint to return dummy response
 - Also added swagger dependency for generating api specs
 - Added log format as per DIGIT guidelines
 - jayant|jitendra

### CODE REVIEW

File: digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/Individual.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Individual {
+    @JsonProperty("individualId")
+    private String individualId;
+
+    @JsonProperty("name")
+    private String name;
```
Comment (egov-joy) [inline digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/Individual.java:20]: Mrunal mentioned they might want to capture individual fields for fields like name (familyName, givenName, additionalName) and address etc.

File: digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/CampaignData.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonSubTypes({ @JsonSubTypes.Type(value = HouseholdRegistration.class,
+        name = "householdRegistration"),
+        @JsonSubTypes.Type(value = Delivery.class, name = "delivery")})
```
Comment (egov-joy) [inline digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/CampaignData.java:15]: We should also have Registration and IndividualRegistration as possible types?

Comment (jayantp-egov) [reply digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/CampaignData.java:15]: We will add these when we implement the actual Registration service. For the first use case, the scope is to only log the request in Registration service.

File: digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/Individual.java (hunk 1 above)
Comment (jayantp-egov) [reply digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/Individual.java:20]: Made the suggested changes.

**Review by egov-joy — APPROVED** [review]

**Pull request merged**